pkg/fileservice/mem_cache.go

```diff
@@ -49,8 +48,7 @@ func (m *MemCache) Read(
 ) (
 	err error,
 ) {
-	_, span := trace.Start(ctx, "MemCache.Read")
-	defer span.End()
+	metric.FsMemCacheReadCounter.Inc()
 
 	numHit := 0
 	defer func() {
```

Comment:
here `stats` already do the count, but no any report output.  is  it better to combine metric with `m.stats` ?

Reply:
Yes. That makes sense. I will refactor the code to include these changes.

Reply:
Please find the changes requested in the new PR. https://github.com/matrixorigin/matrixone/pull/8351